modules/tensor_mechanics/test/tests/notched_plastic_block/biaxial_abbo.i

```diff
@@ -0,0 +1,237 @@
+# Uses an Abbo et al smoothed version of Mohr-Coulomb (via TensorMechanicsPlasticMohrCoulomb and ComputeMultiPlasticityStress) to simulate the following problem.
+# A cubical block is notched around its equator.
+# All of its outer surfaces have roller BCs, but the notched region is free to move as needed
+# The block is initialised with a high hydrostatic tensile stress
+# Without the notch, the BCs do not allow contraction of the block, and this stress configuration is admissible
+# With the notch, however, the interior parts of the block are free to move in order to relieve stress, and this causes plastic failure
+# The top surface is then pulled upwards (the bottom is fixed because of the roller BCs)
+# This causes more failure
+[Mesh]
+  type = GeneratedMesh
+  dim = 3
+  nx = 9
+  ny = 9
+  nz = 9
+  xmin = 0
+  xmax = 0.1
+  ymin = 0
+  ymax = 0.1
+  zmin = 0
+  zmax = 0.1
+[]
+
+[MeshModifiers]
+  [./block_to_remove_xmin]
```

Comment:
We are thinking about implementing an input file formatter (much like clang-format). The input file formatter will probably drop the "dots" and "dot slashes". However, it'll also be customizable per module/application. Start thinking about what you want for your modules and we'll eventually make a sweeping commit to reformat then enforce a standard across modules and the framework. In the framework we are starting to drop the dots and slashes for what it's worth. No enforcement yet though.

Reply:
I think for PorousFlow, we'd probably like to go with the most modern version, because then there's less confusion when people read framework tests and are comparing them with porousflow tests.  Do you agree, @cpgr ?

However, for Richards, i would very much like to retain exactly the same file syntax.  This is because it's a "legacy" module, still used by a small number of projects, but they're important projects in which being able to run years-old input files is important.

Reply:
Yep, making PorousFlow match the framework would be the best option